src/main/java/io/github/moulberry/notenoughupdates/NotEnoughUpdates.java

```diff
@@ -236,6 +237,9 @@ public void preinit(FMLPreInitializationEvent event) {
 		SlotLocking.getInstance().loadConfig(new File(neuDir, "slotLocking.json"));
 		ItemPriceInformation.init(new File(neuDir, "auctionable_items.json"), gson);
 
+		if (config != null)
+			if (config.mining.powderGrindingTrackerResetMode == 2) OverlayManager.powderGrindingOverlay.loadData(new File(neuDir, "powderGrinding.json"));
```

Comment:
why are you making a new file instead of using HiddenProfileSpecific in the main config?

Reply:
Should I store all information in there or only save and load when necessary?

Reply:
You can move your settings in there and delete the fields and the file handling entirely. You don't need to handle save and load at all that way, Plus one fewer file.